.circleci/config.yml

```diff
@@ -166,13 +185,17 @@ workflows:
   validate:
     when: << pipeline.parameters.run_pr_validator >>
     jobs:
-      - job_pr-validator:
+      - job_pr-validator-win32:
```

Comment:
We should add a slack Notification post step here too.

Reply:
I will - once we have fixed all the tests otherwise this will continue to ping us that the test failed.